app.properties

```diff
@@ -1,12 +1,68 @@
-project.name:
-project.repo:
-project.sdgs:
-project.app.url:
-project.codacy.url:
-
-project.team.members:
-project.team.slackname:
-
-project.fb.opensource:
-
-program.cohort:1
\ No newline at end of file
+# about the product
+# ================= 
+# name of the product 
+name: SmeVest

```

Comment:
@Yasholma do you know why we have this ```^M```?

Reply:
no

Reply:
Is it version?